src/main/java/pro/sky/EmployeeService.java

```diff
@@ -0,0 +1,46 @@
+package pro.sky;
+
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+@Service
+public class EmployeeService {
+    private static final int maxEmployees = 10;
+    private static final List<Employee> employees = new ArrayList<>();
```

Comment:
```suggestion
    private final int maxEmployees = 10;
    private final List<Employee> employees = new ArrayList<>();
```
поля не должны быть статическими